handlers.c

```diff
@@ -795,27 +809,91 @@ bool handler__decinc(globals_t * vars, char **argv, unsigned argc)
     {
         m = (argc == 1) ? MATCHDECREASED : MATCHDECREASEDBY;
     }
-    else
+    else if (m == MATCHUNSPECIFIED)
     {
         show_error("unrecogised match type seen at decinc handler.\n");
         return false;
     }
 
+    bool rval = do_search ( m, vars, pval, argv, argc );
+
+    if ( m == MATCHRANGE )
+        free(pval);
+
+    return rval;
+}
+
+static bool try_get_range_values ( const char * arg, uservalue_t ** pval_out ) 
+{
+    char * argv_copy = strdupa(arg);
+
+    if (!argv_copy)
+        return false;
+
+    char * token = strtok(argv_copy, "..");
+
+    uservalue_t * pval = NULL;
+
+    if (pval_out)
+    {
+        pval = malloc(sizeof(uservalue_t)*2);
```

Comment:
Better clearly show that this is multiplication. Put spaces before and after '\*'.